src/test/users-query-test.ts

```diff
@@ -47,14 +53,16 @@ describe('users query', () => {
   let token: string;
   let usersToSave: CreateUserInputData[];
   let users;
+  let qtdUsersToSave;
 
   beforeEach(async () => {
     userRepository = AppDataSource.getRepository(User);
     await userRepository.clear();
     usersToSave = [];
     users = [];
+    qtdUsersToSave = 50;
```

Comment:
```suggestion
    quantityOfUsersToSave = 50;
```

No geral é legal evitar abreviações, porque não tem nada mais claro do que as palavras escritas por completo, certo?